pytorch_blade/src/torch-mlir/lib/Conversion/TorchToMhlo/TorchToMhlo.cpp

```diff
@@ -543,15 +544,10 @@ class ConvertAtenDivOp : public OpConversionPattern<AtenOpT> {
     auto lhsTy = lhs.getType().dyn_cast<TensorType>();
     Value rhs = adaptor.other();
     auto rhsTy = rhs.getType().dyn_cast<TensorType>();
-
+    auto lhsElemTy = lhsTy.getElementType();
     if (!lhsTy)
```

Comment:
Why place this statement before checking `lhsTy == nullptr`

Reply:
fix